agent/scripts/fetch-winners-archive.js

```
@@ -0,0 +1,67 @@
+'use strict';
+
+const fs = require('fs');
```

ESLint

[error] 'require' is not defined.

(no-undef)